templates/solidCommonVocabDependent/javascript/package.hbs

```diff
@@ -32,6 +32,7 @@
   },{{/if}}{{/if}}
   "dependencies": {
     "@inrupt/solid-common-vocab": "{{solidCommonVocabVersion}}",
+    "@rdfjs/types": "{{rdfjsTypesVersion}}",

```

Comment:
not sure the types needs to be in the regular js template

Reply:
Yep - 039ac4c